modules/gitops-release-notes-1-13-3.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-13-3_{context}"]
+= Release notes for {gitops-title} 1.13.3
+
+{gitops-title} 1.13.3 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-13.3_{context}"]
+== Errata updates
+
+[id="rhba-2024:10127-gitops-1-13-3-security-update-advisory_{context}"]
+=== RHBA-2024:10127 - {gitops-title} 1.13.3 security update advisory
+
+Issued: 2024-11-20
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHBA-2024:10127[RHBA-2024:10127]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-13-3_{context}"]
+== Fixed issues
+
+* Before this update, when using the dynamic scaling feature, application controller pods consumed high memory and clusters were not distributed properly across shards. This update ensures that clusters are distributed across shards. link:https://issues.redhat.com/browse/GITOPS-5811[GITOPS-5811]
++
+For more information, see xref:../declarative_clusterconfig/sharding-clusters-across-argo-cd-application-controller-replicas.adoc#gitops-argo-cd-dynamic-scaling-in-web-console_sharding-clusters-across-argo-cd-application-controller-replicas[Enabling dynamic scaling of shards in the web console section]
```

Comment:
Just like in the other PR, can you make the following change?

I'm not going to hold up the merge of this PR, since I see the release date was yesterday. But please fix this in a future PR if you don't mind!

```suggestion
For more information, see xref:../declarative_clusterconfig/sharding-clusters-across-argo-cd-application-controller-replicas.adoc#gitops-argo-cd-dynamic-scaling-in-web-console_sharding-clusters-across-argo-cd-application-controller-replicas[Enabling dynamic scaling of shards in the web console].
```

Reply:
Fixed.